scripting/include/srccoop/playerpatch.inc

```diff
@@ -612,6 +615,23 @@ public Action Hook_PlayerTakeDamage(int victim, int &attacker, int &inflictor, f
 	return Plugin_Continue;
 }
 
+// Fixes hit registration being off by 1 tick.
+// Credits to Xutax_Kamay (https://forums.alliedmods.net/showthread.php?t=315405)
+public MRESReturn Hook_PlayerWeaponShootPosition_Post(int iClient, Handle hParams)
```

Comment:
hParams can be removed if there are none

Reply:
Renamed to `hReturn`.